article/structure.py

```diff
@@ -246,10 +264,8 @@
     ALL_ARTICLES,
 ]
 ALL_GROUPS_DICT = {group.key: group for group in ALL_GROUPS}
-ALL_GROUPS_ARTICLES_SETS = {
-    group.key: frozenset([article.uuid for article in group.articles]) for
-    group in ALL_GROUPS
-}
+ALL_ARTICLES_DICT = {article.uuid: article for
```

Comment:
not a fan of using type names in vars. Would go with just `ALL_ARTICLES` or `ARTICLE_PER_UUID`, but it's not that big of a deal